editoast/src/views/rolling_stocks.rs

```diff
@@ -320,7 +320,7 @@ async fn create_compound_image(
 }
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
```

Comment:
```suggestion
mod tests {
```
No need if you use fixtures in your test